src/main/interface/msp.c

```diff
@@ -1185,6 +1185,7 @@ static bool mspProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst)
         break;
 
     case MSP_FILTER_CONFIG :
+        sbufWriteU8(dst, gyroConfig()->gyro_soft_pre_lpf_hz);
```

Comment:
This will break MSP compatibility ...

Reply:
Forgive my ignorance I don't know what MSP compatibility means or how I would break it with this code, by all means let me know how this should be fixed.   I was intending to change all gyro set points to u16 so as to increase the upper limit.  This certainly needed for the 'higher' of the two, the 'standard' low-pass could stay limited to 255hz I guess.

Reply:
Standard way is to add new config options to end of packet and handle missing ones gracefully.

As implemented in this PR, configurator must be upgraded to handle FILTER_CONFIG (and version mismatch will result in invalid values for all filters ... )

Reply:
That line need to be a empty write and at the end of the message the "new" value need to be added :)
I think those will be easy and fast changes once we have a vision what do to. Currently there are 3 different proposals that change the filtering stuff in a very similar way :)
Once that is more straighten out it should be added to msp.

Reply:
Or just skip MSP for experimental features ...